src/components/app/list-new/Constants.ts

```diff
@@ -29,6 +29,15 @@ export const APPLIST_EMPTY_STATE_MESSAGING = {
     noAppsFoundInfoText: `We couldn't find any matching applications.`,
 }
 export const ClearFiltersLabel = 'Clear filters'
+export const DefaultAppNote =`## Describe this application
+
```

Comment:
We should use some other option for formatting adding new line like this is not the correct way

Reply:
done